openbb_platform/extensions/technical/openbb_technical/relative_rotation.py

```diff
@@ -0,0 +1,546 @@
+"""Relative Rotation Model."""
+
+# pylint: disable=too-many-arguments, too-many-instance-attributes, too-many-locals, too-few-public-methods, unused-argument
+
+import contextlib
+from typing import Any, Dict, List, Literal, Optional, Tuple, Union
+
+import numpy as np
+from openbb_core.app.model.obbject import OBBject
+from openbb_core.app.utils import basemodel_to_df, convert_to_basemodel, df_to_basemodel
+from openbb_core.provider.abstract.data import Data
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.abstract.query_params import QueryParams
+from pandas import DataFrame, Series, to_datetime
+from pydantic import Field, field_validator
+
+
+def absolute_maximum_scale(data: Series) -> Series:
+    """Absolute Maximum Scale Normaliztion Method"""
+    return data / data.abs().max()
+
+
+def min_max_scaling(data: Series) -> Series:
+    """ "Min/Max ScalingNormalization Method"""
+    return (data - data.min()) / (data.max() - data.min())
+
+
+def z_score_standardization(data: Series) -> Series:
+    """Z-Score Standardization Method."""
+    return (data - data.mean()) / data.std()
+
+
+def normalize(data: DataFrame, method: Literal["z", "m", "a"] = "z") -> DataFrame:
```

Comment:
Should this functions (normalize, standard_deviation, calculate_relative_strength_ratio, calculate_momentum, etc.) be inside the class or if not and the intention is to use them as utilities in the future, let's move them to another file.

Reply:
They are excluded specifically from the class because they should not get bound to it. Those functions, however, are specifically tailored to this use. They are meant to serve this function only.